ovn-fake-multinode-utils/playbooks/deploy-minimal.yml

```diff
@@ -3,9 +3,8 @@
   become: true
   tasks:
   - name: Start openvswitch
-    systemd:
-      name: openvswitch
-      state: started
+    shell: |
+      systemctl start openvswitch || systemctl start openvswitch-switch
```

Comment:
Nit: what if we use the `ansible_os_family` (https://docs.ansible.com/ansible/2.6/user_guide/playbooks_conditionals.html#ansible-os-family) conditional here and different tasks per OS family?

Reply:
Using `ansible_os_family` would also allow us to use a a single `install-dependencies.yml` playbook.

Reply:
I was wondering if there was a simple way of doing this with Ansible, and it looks like there is, I'll have a go at it. Thx for pointing it out!

Reply:
Done